common/workunit/workunit.cpp

```diff
@@ -13015,15 +13016,15 @@ extern WORKUNIT_API void addTimeStamp(IWorkUnit * wu, StatisticScopeType scopeTy
     wu->setStatistic(queryStatisticsComponentType(), queryStatisticsComponentName(), scopeType, scopestr, kind, NULL, getTimeStampNowValue(), 1, 0, StatsMergeAppend);
 }
 
-extern WORKUNIT_API double calculateThorCost(__int64 timeNs, unsigned clusterWidth)
+extern WORKUNIT_API double calculateThorCost(double elapsedTime, unsigned clusterWidth)
```

Comment:
I think 'elapsedTime' units are seconds? But would be clearer if elapsedTimeSecs